src/pages/extending-agent-app/products-sdk/agent-app-sdk/index.mdx

```diff
@@ -443,6 +443,75 @@ widget.setReportsFilters({ tag: ['chatbot'] });
 
 </CodeSample>
 
+## Settings widgets
+
+If you want to connect a Settings widget to the LiveChat Agent App, you should use the `createSettingsWidget` function. It returns a promise resolving to a Settings widget instance.
+
+<Code>
+<CodeSample path={'SETTINGS WIDGETS'}>
+
+```js
+import { createSettingsWidget } from "@livechat/agent-app-sdk";
+
+createSettingsWidget().then(widget => {
+  // build your logic around the widget
+});
+```
+
+</CodeSample>
+</Code>
+
+### Events
+
+#### `page_data`
+
+Emitted when widget in initialized. The handler will get the main window page data object as an argument.
+
+Listen for the page data changes:
+
+<Code>
+<CodeSample path={'PAGE_DATA'}>
+
+```js
+widget.on("page_data", pageData => {
+  // read the page data
+});
+```
+
+</CodeSample>
+</Code>
+
+### Methods
+
+- [Get page data](#get-page-data)
+- [redirect](#redirect)
+
+#### Get page data
+
+Gets the main window page data recorded most recently. Returns the `IPageData` object, which is identical to the one emitted by the `page_data` event or null (if no data were registered).
+
+<CodeSample path={'GET PAGE DATA'}>
+
+```js
+widget.getPageData();
+```
+
+</CodeSample>
+
+#### Redirect
+
+Redirects using the main window. Calling this method will send postmessage to Agent App, witch will be handled there. After that, redirect using window object is performed.
```

Comment:
```suggestion
Redirects using the main window. Calling this method will send `postmessage` to the LiveChat App, which will be handled there. After that, a redirect using a window object is performed.
```

Reply:
what purpose does `postmessage` serve here? Shouldn't it be `postMessage` as well?

Reply:
It's an util allowing the iframed app communicate with parent (Agent App)
https://github.com/livechat/products-sdk/blob/67928fbdfbc20a85b98a75645464c60dabb8f381/packages/widget-core-sdk/src/connection/driver.ts#L29

Reply:
9304fdfe99adbde9b902385cc7cdae59b4d4bc4d - fixed copy